themes/default/assets/sass/_api-nodejs.scss

```
@@ -36,15 +36,15 @@ h3.api {
 ul.api {
     @apply list-none p-0 m-0 overflow-hidden;
 
-    columns: 3 220px;
+    columns: 2 330px;
```

Rather than hard-code a width for these, we might as well just explicitly say, "it's one column by default, two at desktop width and up": 

```suggestion
    columns: 1;

    @screen xl {
        columns: 2;
    }
```

https://user-images.githubusercontent.com/274700/118325205-fc4f4380-b4b7-11eb-844b-116d7bd95254.mp4

---

oh okay, i was trying to align with what was already there